server/conf/candlepin.conf.template

```diff
@@ -32,7 +32,7 @@ candlepin.importer.fail_on_unknown=false
 
 <% if (candlepin.external_broker) { %>\
 candlepin.audit.hornetq.embedded=false
-candlepin.audit.hornetq.broker_url=tcp://localhost:61617
+candlepin.audit.hornetq.broker_url=tcp://localhost:61617?sslEnabled=true&trustStorePath=/var/lib/artemis/candlepin/certs/artemis-client.ts&trustStorePassword=securepassword&keyStorePath=/var/lib/artemis/candlepin/certs/artemis-client.ks&keyStorePassword=securepassword
```

Comment:
I feel like rather than the parameter soup on the URL we should build this internally using other config parameters. Maybe this will work for the short term (i.e. master with knowledge of the coming changes in the feature branch), but long-term it's kind of a mess.

Reply:
I've split these into multiple parameters.